pkg/kubelet/cm/cpuset/cpuset.go

```diff
@@ -162,20 +162,17 @@ func (s CPUSet) Difference(s2 CPUSet) CPUSet {
 	return s.FilterNot(func(cpu int) bool { return s2.Contains(cpu) })
 }
 
-// ToSlice returns a slice of integers that contains all elements from
-// this set.
-func (s CPUSet) ToSlice() []int {
-	result := make([]int, 0, len(s.elems))
-	for cpu := range s.elems {
-		result = append(result, cpu)
-	}
+// List returns a slice of integers that contains all elements from
+// this set. The list is sorted.
+func (s CPUSet) List() []int {
```

Comment:
would it be more logical to call it SortedList? It's not obvious that List() result is sorted without knowing that there is an UnsortedList().

Reply:
I am only naming it List() because this mirrors the 'set' API.

In 'set', conversions to slice are done also, but with different names:

ToSliceNoSort() -> UnsortedList()
ToSlice() -> List()

(see https://github.com/kubernetes/kubernetes/pull/113744/commits/bc95376e283ce179eb42bf4b2081f3d3d9b99038 for the commit that does this)